### ISSUE TITLE
Invalid argument propertyAccessor.setValue on IE 11

### ISSUE THREAD

**Issue by Jenselme** [aurelia/binding#414]
Hi,

I just tried my Aurelia application on IE11 and discovered it is broken. I get a `SCRIPT87: Invalid argument.` error [here](https://github.com/aurelia/binding/blob/69d54dbebafe87738f89b8ac4109921544d202cb/dist/commonjs/aurelia-binding.js#L3460). It works fine in all other browsers (Firefox and Chrome).

I tried to look at the problem, by adding this line before the setting of the value: ` console.log('nrsauternet', value, obj, propertyName, Object.keys(obj), typeof obj, typeof value);`. Here is what I get:
![screenshot from 2016-05-27 23-14-21](https://cloud.githubusercontent.com/assets/6643536/15621397/dd649a7a-2460-11e6-9bee-71dccf7ce65a.png)

Which looks good to me. I don't know what is causing this.

**Comment by jdanyow** [comment]
when the error occurs, what is the value of `obj` and `propertyName` at the [line of code](https://github.com/aurelia/binding/blob/69d54dbebafe87738f89b8ac4109921544d202cb/dist/commonjs/aurelia-binding.js#L3460) you mentioned

**Comment by Jenselme** [comment]
As shown in the capture:
- `propertyName` is `textContent` (string)
- `obj` is some king of object called `EmptyTextNode` although I have also seen the error with a `Text` object.

Do you need more information? If so, can you be a little more specific on what you need?

**Comment by jdanyow** [comment]
any chance you could reproduce this issue using plunker:  http://plnkr.co/edit/1cBskxDsHcMP1hLmedyw?p=preview

**Comment by Jenselme** [comment]
I managed to dug into it today and I found the view that caused problem in my app. It came from this code:

```html
    <p class="centered-important"
       t="[html]game.play.whose_turn_message"
       t-params.bind="{playerName: currentPlayerName}">
        C'est le tour de<br>
        <strong>${playerName}</strong>
    </p>
```

The injected translation by aurelia-i18n is `"It is the turn of <br /><strong>{{playerName}}</strong>"`. If I remove `t-params.bind` it works (but the message is not correctly translated).

To make it work without errors, I removed `<strong>${playerName}</strong>` (legacy from when I had not yet translations). I guess that this was confusing IE.

I tried to reproduce [in this commit](https://github.com/Jenselme/skeleton-navigation/commit/24863bfbc4da098ea603fe08354bde4622845483) on a small app based on the skeleton. The result can be tested here: http://testaot.jujens.eu/ However, it works fine on IE11 even if it looks very similar to what my code was. Can see something different that might explain that this small example works but not my app?

Anyway, I don't know if it is a real bug in aurelia-binding or just bad luck from IE.
